l10n_eu_product_adr/views/product_template_views.xml

```diff
@@ -19,24 +19,18 @@
                     name="adr"
                     string="Dangerous Goods"
                     groups="l10n_eu_product_adr.group_adr_goods_user"
-                    attrs="{'invisible': ['|', ('type', '!=', 'product'), '&amp;', ('is_dangerous', '=', False), ('adr_goods_on_variants', '=', False)]}"
+                    invisible="not (is_storable and (is_dangerous or adr_goods_on_variants))"
```

Comment:
are we sure this is the same ?
I would have written this `not is_storable or (is_dangerous and adr_goods_on_variants)`

Reply:
no, it will be: `not is_storable or (not is_dangerous and not adr_goods_on_variants)`

Reply:
Thanks for the change it looks better IMO